build.sbt

```diff
@@ -211,6 +212,43 @@ lazy val testStdLayout = Project("daffodil-test-stdLayout", file("test-stdLayout
   .dependsOn(tdmlProc % "test")
   .settings(commonSettings, nopublish)
 
+/* Workaround: certain reflection (used by JAXB) isn't allowed by default in JDK 17:
+ * https://docs.oracle.com/en/java/javase/17/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B
+ *
+ * While we can handle this JVM quirk at build time, at runtime we won't know
+ * a user's JVM version. We'll provide documentation and an extension setting
```

Comment:
We do compile daffodil but then run it with a variety of JVM versions, so it's possible this needs to be a runtime thing. 
E.g., are these options even allowed on earlier JVMs? 

@stevedlawrence please look this over.

Reply:
I also accidentally type "optinos" all the time :)